test/e2e/storage/vsphere/vsphere_statefulsets.go

```diff
@@ -146,9 +148,13 @@ var _ = utils.SIGDescribe("vsphere statefulset [Feature:vsphere]", func() {
 					vSpherediskPath := getvSphereVolumePathFromClaim(client, statefulset.Namespace, volumespec.PersistentVolumeClaim.ClaimName)
 					framework.Logf("Verify Volume: %q is attached to the Node: %q", vSpherediskPath, sspod.Spec.NodeName)
 					// Verify scale up has re-attached the same volumes and not introduced new volume
-					framework.ExpectEqual(volumesBeforeScaleDown[vSpherediskPath] == "", false)
+					if volumesBeforeScaleDown[vSpherediskPath] == "" {
+						framework.Failf("Volume: %q is not attached to the Node: %q", vSpherediskPath, sspod.Spec.NodeName)
```

Comment:
This is the same failure message as below. Perhaps change to "was not attached to the Node before scale down"?

Reply:
I have updated the message, PTAL.